### PR TITLE
Updated Polish localization.

### PR DESCRIPTION
**Pull request by Piter432**
However, I'm not sure that I should leave “**__**MSG_extension_name__” and “**__**MSG_extension_desc__” or not.

### CODE REVIEW

File: _locales/pl/messages.json [hunk 1]
```diff
@@ -135,15 +135,27 @@
     "description": "Tab title for the image properties pop-up. Refers to data embedded within an image, such as Exif data."
   },
   "prop_header_width": {
-    "message": "80px",
+    "message": "80 pikseli",
```
Comment (ChaosinaCan) [inline _locales/pl/messages.json]: This actually doesn't need to be translated. It's so you can increase the size of the header column if you need to:

image

It gets placed directly into the CSS, so it should stay "px".

**Comment by ChaosinaCan** [comment]
I don't think it hurts anything to leave them in, but __MSG_extension_name__ and __MSG_extension_desc__  don't need to be there.

See my comment on the change though. One of the fields is actually a CSS value. The number should be adjusted if necessary, but it shouldn't be translated.

**Comment by Piter432** [comment]
Ah, okay. I wasn't sure is that CSS code or not :stuck_out_tongue: I'll fix that.

**Comment by Piter432** [comment]
Fix published.

**Pull request merged**